distribution/ddf-common/src/main/resources/etc/system.properties

```diff
@@ -23,71 +23,133 @@
 #
 
 #START DDF SETTINGS
-# Set the keystore and truststore Java properties
+
+#
+# Keystore and Truststore Java Properties
+#
 javax.net.ssl.keyStore=etc/keystores/serverKeystore.jks
 javax.net.ssl.keyStorePassword=changeit
 javax.net.ssl.trustStore=etc/keystores/serverTruststore.jks
 javax.net.ssl.trustStorePassword=changeit
 javax.net.ssl.keyStoreType=jks
 javax.net.ssl.trustStoreType=jks
 
-#force java to run in headless mode for when the server doesn't have a display device
-java.awt.headless=true
 
-# Set the global url properties
+#
+# Headless Configuration
+#
+
+# Forces java to run in headless mode for when the server doesn't have a display device
+#java.awt.headless=true
+
+
+#
+# Global URL Properties
+#
 org.codice.ddf.system.protocol=https://
 org.codice.ddf.system.hostname=localhost
 org.codice.ddf.system.httpsPort=8993
 org.codice.ddf.system.httpPort=8181
 org.codice.ddf.system.port=8993
 org.codice.ddf.system.rootContext=/services
 
-# Set the system information properties
+
+#
+# System Information Properties
+#
 org.codice.ddf.system.siteName=${sitename.default}
 org.codice.ddf.system.siteContact=
 org.codice.ddf.system.version=${project.version}
 org.codice.ddf.system.organization=${organization.name}
 
-# Solr client configuration
-# Supported clients: HttpSolrClient CloudSolrClient EmbeddedSolrServer
+
+#
+# Solr Client Configuration
+#
+
+# Uncomment the properties based on the desired configuration:
+
+# For Http Solr Client:
 solr.client = HttpSolrClient
 solr.http.url = ${org.codice.ddf.system.protocol}${org.codice.ddf.system.hostname}:${org.codice.ddf.system.httpsPort}/solr
+solr.data.dir = ${karaf.home}/data/solr
+
+# For Cloud Solr Client:
+# solr.client = CloudSolrClient
 # solr.data.dir = ${karaf.home}/data/solr
```

Comment:
Dan is right that the Solr Cloud client factory does not currently use that system property.  It does default to the value you listed but the only way to change the location used by that factory is to change `karaf.home`.  HTTP and embedded factories will override the default if `solr.data.dir` is set.  The cloud client factory does not do that currently.

I think it is fine either way.  Solr Cloud will not store data in that default location.  It just writes out the files there before uploading to Solr Cloud.  But it would not hurt to make it consistent either.